src/app/services/sync/item.service.ts

```diff
@@ -8,7 +8,7 @@ import {
 } from './graphql.queries';
 import { AllTasks, Task, MutationType } from './types';
 import { VoyagerService } from './voyager.service';
-import { VoyagerClient, createOptimisticResponse } from '@aerogear/voyager-client';
+import { VoyagerClient, createOptimisticResponse, OfflineStore } from '@aerogear/voyager-client';
```

Comment:
all metions of VoyagerClient need to change to `ApolloOfflineClient` once the sdk pr lands.